droid-hal-device.inc

```diff
@@ -381,6 +381,17 @@ cat <<EOF >> $HDRS/android-config.h.new
 /* Added by Droid HAL packaging for %{rpm_device} */
 %{?android_config}
 EOF
+# Detect QCOM_BSP and add the define into android-config.h if needed
+if (grep -q '^TARGET_USES_QCOM_BSP := true' %{android_root}/device/%{vendor}/*/BoardConfig*.mk || \
+    (grep -q 'PLATFORM_VERSION := 5.' %{android_root}/build/core/version_defaults.mk && \
+     grep -q '^BOARD_USES_QCOM_HARDWARE := true' %{android_root}/device/%{vendor}/*/BoardConfig*.mk)); then
+
+    echo Found QCOM_BSP
+    cat <<EOF >> $HDRS/android-config.h.new
+/* Added by automatic QCOM_BSP detection */
+#define QCOM_BSP 1
```

Comment:
This is a macro from Android headers, thus it needs to stay that name to affect places like https://github.com/CyanogenMod/android_hardware_libhardware/blob/cm-12.0/include/hardware/gralloc.h#L327